app/templates/public/login.html

```diff
@@ -6,22 +6,25 @@
 {% block main %}
 
 <!--Login Card-->
-  <div class="container container text-center w-50">
-     <div class="container">
+<div class="container text-center w-50">
+    <div class="container">
         <h1>Login 🎻</h1>
     </div>
     <div class="card text-center mb-3 shadow p-3 mb-5 bg-white rounded">
-      <div div class="card-body">
-      {% with messages = get_flashed_messages() %}
-        {% if messages %}
-          {% for message in messages %}
-           <p>{{ message }}</p>
-          {% endfor %}
-        {% endif %}
-      {% endwith %}
-        {{ wtf.quick_form(form, novalidate=True, button_map={"submit": "outline-dark"}) }}
-      </div>
+        <div div class="card-body">
+            {% with messages = get_flashed_messages() %}
+            {% if messages %}
+            {% for message in messages %}
+            <p>{{ message }}</p>
+            {% endfor %}
+            {% endif %}
+            {% endwith %}
+            {{ wtf.quick_form(form, novalidate=True, button_map={"submit": "outline-dark"}) }}
+            {% if not logged_in %}
+            <a href="{{ url_for('register') }}">Register</a>
+            {% endif %}
```

Comment:
Would you want to indent the the blocks here for readability?
```suggestion
            {% with messages = get_flashed_messages() %}
              {% if messages %}
                {% for message in messages %}
                  <p>{{ message }}</p>
                {% endfor %}
              {% endif %}
            {% endwith %}
            {{ wtf.quick_form(form, novalidate=True, button_map={"submit": "outline-dark"}) }}
            {% if not logged_in %}
              <a href="{{ url_for('register') }}">Register</a>
            {% endif %}
```